pycbc/filter/matchedfilter.py

```diff
@@ -1831,16 +1831,42 @@ def followup_event_significance(ifo, data_reader, bank,
     onsource_end += coinc_threshold
 
     # Calculate how much time needed to calculate significance
-    trim_pad = (data_reader.trim_padding * data_reader.strain.delta_t)
-    bdur = int(lookback + 2.0 * trim_pad + length_in_time)
-    if bdur > data_reader.strain.duration * .75:
-        bdur = data_reader.strain.duration * .75
+    trim_pad = data_reader.trim_padding * data_reader.strain.delta_t
+    bdur = int(lookback + 2 * trim_pad + length_in_time)
```

Comment:
I do not actually understand why we truncate to integer here, since `bdur` is a time in seconds. Should I remove this?

Reply:
You may be right.

Reply:
Ok, will remove then.